### ISSUE TITLE
Getting Invalid ARIA attribute value accessibility error in detailsList header component

### ISSUE THREAD

**Issue by mdmdakbar** [OfficeDev/office-ui-fabric-react#3484]
**Issue** -
Getting following accessibilty error in each details list header cell:
Invalid ARIA attribute value: aria-describedby="header97-name-tooltip"

The dom element corresponding to id passed in aria-describedby does not exist and keros for web is giving the above error.

Below is the method in **DetailsHeader.tsx** where each column header is rendered -

private _onRenderColumnHeaderTooltip(tooltipHostProps: ITooltipHostProps, defaultRender?: IRenderFunction<ITooltipHostProps>) {
    return (
      span className={ tooltipHostProps.hostClassName }>
        { tooltipHostProps.children }
      /span >
    );
  }
Here tooltipHostProps.children is using aria-describedby property but the id corresponding to it is not set anywhere.

Are you willing to submit a PR to fix? No

Products/sites affected: VSTS

Attaching the snapshot of one of the usages of detailsList in  VSTS.
![capture1](https://user-images.githubusercontent.com/19491487/33431083-066f8c68-d5f9-11e7-9195-e7443f318a27.PNG)

This is present across all usages of detailsList in VSTS.

**Comment by KevinTCoughlin** [comment]
@mdmdakbar, closing this as it should be fixed by https://github.com/OfficeDev/office-ui-fabric-react/pull/4487 and here's a related issue https://github.com/OfficeDev/office-ui-fabric-react/issues/4372

Please re-open if that's not the case. Thank you for the report!
